vignettes/articles/01-setup-project-directories.Rmd

```diff
@@ -66,7 +61,9 @@ A recommended structure for the project-agnostic data would look like this:
     * `prewrangled_capacity_factors_WEO_2020.csv`
```

Comment:
we will use WEO2019 cap factors for now, to be consistent with price data and scenario data.

Reply:
changed to 2019